frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -45,9 +47,7 @@ const getDefaultServingRuntime = (servingRuntimesConfig?: ConfigMapKind): Servin
   if (defaultServingRuntime) {
     return defaultServingRuntime;
   }
-  console.error(
```

Comment:
Removed this, talked UX about the visual cue but I think it can be applied to the console too